tidb-in-kubernetes: add offline pump document (#1555)
pingcap/docs: dev/tidb-in-kubernetes/maintain/backup-and-restore.md

Incremental backup uses [TiDB Binlog](/dev/reference/tidb-binlog-overview.md) to collect binlog data from TiDB and provide real-time backup and replication to downstream platforms.

For the detailed guide of maintaining TiDB Binlog in Kubernetes, refer to [TiDB Binlog](/dev/tidb-in-kubernetes/maintain/tidb-binlog.md).

> ### Decrease the capability of Pump

Reviewer: Use `Scale down` instead of `Decrease the capability of`?

> To decrease the capability of Pump, make Pump nodes offline and then run the `helm upgrade` command to delete Pump pods.

Reviewer 2: ```suggestion
To scale Pump in, you decrease the number of Pump nodes by taking the nodes offline one at a time and running the `helm upgrade` command to delete the corresponding Pump Pod. The steps are as follows:
```
"Pod" needs to be capitalized. ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/

> 1. Make Pump nodes offline from the TiDB cluster

Reviewer 2: ```suggestion
1. Make the Pump node offline from the TiDB cluster
```

>     Suppose there are 3 Pump nodes, and you want to get the third node offline and modify `<ordinal-id>` to `2`, run the following command. `<version>` is the current version of TiDB.

Reviewer 2: ```suggestion
    Suppose there are 3 Pump nodes, and you want to get the third node offline and modify `<ordinal-id>` to `2`, run the following command (`<version>` is the current version of TiDB).
```

    {{< copyable "shell-regular" >}}

    ```shell
    kubectl run offline-pump-<ordinal-id> --image=pingcap/tidb-binlog:<version> --namespace=<namespace> --restart=OnFailure -- /binlogctl -pd-urls=http://<release-name>-pd:2379 -cmd offline-pump -node-id <release-name>-pump-<ordinal-id>:8250
    ```

    Then, check the log output of Pump. If Pump outputs `pump offline, please delete my pod`, the state of the Pump node is successfully switched to `offline`.

    {{< copyable "shell-regular" >}}

    ```shell
    kubectl logs -f -n <namespace> <release-name>-pump-<ordinal-id>
    ```

> 2. Delete Pump pods

Reviewer 2: ```suggestion
2. Delete the corresponding Pump Pod
```

>     Modify `binlog.pump.replicas` in the `values.yaml` file to `2` and then, run the following command to delete Pump pods.

Reviewer 2: ```suggestion
    Modify `binlog.pump.replicas` in the `values.yaml` file to `2` and then run the following command to delete the Pump Pod.
```
